cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/worker/TaskWorkerServiceTest.java

```diff
@@ -101,7 +103,7 @@ public void afterTest() {
     }
   }
 
-  private void waitForTaskWorkerToFinish(TaskWorkerService taskWorker) {
+  static void waitForTaskWorkerToFinish(TaskWorkerService taskWorker) {
```

Comment:
Can we instead extract this into a test util class?

Reply:
Done